docker-compose.multi.yaml

```diff
@@ -0,0 +1,310 @@
+version: '2.4'
+services:
+  mysql_org_a:
```

Comment:
This has a lot of duplication, but I couldn't figure out how to run two separate stacks of Mattermost on two different subsets otherwise.

Reply:
Probably @khos2ow could help on this part, he is a Docker mastah